server_app/user/handler.js

```diff
@@ -95,6 +95,34 @@ const resetPassword = async(req, res) => {
   }
 }
 
+const updatePassword = async(req, res) => {
+  try {
+    const mail = /*decipher.decode(*/req.body.mail;
```

Comment:
We need the decode since we use the encode on frontend. It is a way to hide information and avoid some kind of of attacks, not very elegant I know but it we will improve it soon.